### PR TITLE
feat(n-qr-code): adds `type` prop, Customize rendering output by setting `type`, providing two options: `canvas` and `svg`.

### PR DESCRIPTION
**Pull request by winssps**
closes https://github.com/tusen-ai/naive-ui/issues/5743.

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## NEXT_VERSION
+
+`NEXT_VERSION`
+
+### Features
+
+- `n-qr-code` adds `type` prop, Customize rendering output by setting `type`, providing two options: `canvas` and `svg`.
```
Comment (Chieffo2021) [inline CHANGELOG.en-US.md:9]: The changelog should be auto updated by script, no?

**Comment by winssps** [comment]
Note: This code refers to https://github.com/zpao/qrcode.react. Thanks here!

**Comment by xiehuijie** [comment]
代码莫得问题，工作很正常

**Comment by 07akioni** [comment]
Good

**Pull request merged**